config/locales/en.yml

```diff
@@ -2690,6 +2690,9 @@ See the %{link} to find out more about %{sitename}'s features and to start using
       increment: "＋" # U+FF0B Fullwidth Plus Sign
       decrement: "－" # U+FF0D Fullwidth Hyphen-Minus
       in_cart: "%{quantity} in cart"
+    bulk_buy_modal:
```

Comment:
looking at this again, I think it would be better if we add some context to these keys, not just js.bulk_buy_modal byt maybe js.shopfront.bulk_buy_modal or something like this. This is good for organization of the keys and also to give some context to translators in transifex.